controllers/slice/namespaces.go

```diff
@@ -417,25 +416,25 @@ func (r *SliceReconciler) deleteAnnotationsAndLabels(ctx context.Context, slice
 				delete(labels, controllers.ApplicationNamespaceSelectorLabelKey)
 			}
 		}
-		podannotations := deploy.Spec.Template.ObjectMeta.Annotations
+		podannotations := pod.ObjectMeta.Annotations
 		if podannotations != nil {
 			v, ok := podannotations["ns.networkservicemesh.io"]
 			if ok && v == "vl3-service-"+slice.Name {
 				delete(podannotations, "ns.networkservicemesh.io")
 			}
 		}
-		deployannotations := deploy.ObjectMeta.GetAnnotations()
-		if deployannotations != nil {
-			_, ok := deployannotations["kubeslice.io/status"]
+		statusannotations := pod.ObjectMeta.GetAnnotations()
```

Comment:
why the name "statusannotations" ?

Reply:
Previously it was deployannotations so this annotation is present in pod annotations so named it as statusannotations